modules/openapi-generator/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -37,7 +37,7 @@ namespace {{packageName}}.Client
     internal class CustomJsonCodec : IRestSerializer, ISerializer, IDeserializer
     {
         private readonly IReadableConfiguration _configuration;
-        private static readonly string _contentType = "application/json";
+        private static ContentType _contentType => RestSharp.ContentType.Json;
```

Comment:
This was a private. You are turning it into a private property.  The generally accepted guidance is that fields are private and named _camelCase while properties are public and PascalCase.

![image](https://github.com/OpenAPITools/openapi-generator/assets/29644329/3cb0e9e1-70cd-43b4-875b-c15977b4805b)